### PR TITLE
8324792: [lworld] remove VarHandle test cases for primitive value class

### PR DESCRIPTION
**Pull request by mlchung**
Clean up test/jdk/java/lang/invoke/VarHandles tests and remove tests for primitive value classes.   Since `.class` is adequate to refer to the type, the template files are now updated to match the mainline with only small change for valhalla.

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [JDK-8324792](https://bugs.openjdk.org/browse/JDK-8324792): [lworld] remove VarHandle test cases for primitive value class (**Enhancement** - P3)

### Reviewers
 * [Roger Riggs](https://openjdk.org/census#rriggs) (@RogerRiggs - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/valhalla.git pull/979/head:pull/979` \
`$ git checkout pull/979`

Update a local copy of the PR: \
`$ git checkout pull/979` \
`$ git pull https://git.openjdk.org/valhalla.git pull/979/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 979`

View PR using the GUI difftool: \
`$ git pr show -t 979`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/valhalla/pull/979.diff">https://git.openjdk.org/valhalla/pull/979.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/valhalla/pull/979#issuecomment-1912867257)

### CODE REVIEW

File: test/jdk/java/lang/invoke/VarHandles/X-VarHandleTestAccess.java.template [hunk 1]
```diff
@@ -506,6 +492,7 @@ public class VarHandleTestAccess$Type$ extends VarHandleBaseTest {
 #end[Bitwise]
     }
 
+
```
Comment (RogerRiggs) [inline test/jdk/java/lang/invoke/VarHandles/X-VarHandleTestAccess.java.template:495]: Extra blank line?

**Review by RogerRiggs — APPROVED** [review]
lgtm

File: test/jdk/java/lang/invoke/VarHandles/X-VarHandleTestAccess.java.template (hunk 1 above)
Comment (mlchung) [reply test/jdk/java/lang/invoke/VarHandles/X-VarHandleTestAccess.java.template:495]: This extra blank line is in the mainline version.

**Pull request closed without merging**